weblate/static/style-bootstrap.css

```diff
@@ -478,12 +478,16 @@ legend {
     width: 35px;
     padding-left: 0;
     padding-right: 0;
+    border-top-right-radius: 4px;
+    border-bottom-right-radius: 4px;
```

Comment:
My guess is that this will go wrong in RTL. Why not using Bootrap button groups to get the proper styling?